python/ray/dataframe/dataframe.py

```diff
@@ -2080,9 +2080,9 @@ def quantile(self, q=0.5, axis=0, numeric_only=True,
 
         Args:
             q (float): 0 <= q <= 1, the quantile(s) to compute
-            axis (int): 0 or 'index' for row-wise,
-                        1 or 'columns' for column-wise
-            interpolation: {'linear', 'lower', 'higher', 'midpoint', 'nearest'}
+            axis (int): 0 or `index` for row-wise,
```

Comment:
It looks like we both changed these lines in different ways. I would prefer the `'`.